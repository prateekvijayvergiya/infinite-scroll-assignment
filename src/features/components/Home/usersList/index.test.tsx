import Component from './'
import makeSnapshot from './../../../testing/makeSnapshot'

jest.mock('@material-ui/core/Grid', () => 'Grid')
jest.mock('@material-ui/core/Typography', () => 'Typography')

jest.mock('./userCard', () => 'UserCard')
jest.mock('../../skeleton', () => 'LoadingSkeleton')
jest.mock('../../common/spacer', () => 'Spacer')

describe('UsersList', () => {
  it('renders correctly', () => {
    makeSnapshot(<Component />)
  })
})
