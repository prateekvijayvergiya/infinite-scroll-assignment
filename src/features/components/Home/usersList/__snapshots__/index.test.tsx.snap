// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersList renders correctly 1`] = `
<Grid
  container={true}
  spacing={3}
>
  <Spacer />
  <LoadingSkeleton />
</Grid>
`;
